docs/guide/setup-examples/julia.md

```diff
@@ -0,0 +1,401 @@
+```
+<snippet>
+    <content><![CDATA[
+"""
+    ${1:bar(x[, y])}
+
+${0:Compute}
+"""
+]]></content>
+    <tabTrigger>docs</tabTrigger>
+    <scope>source.julia</scope>
+</snippet>
+```
+The snippets go into one file per snippet, which I put in `USER\snippets`.
+You can find more snippets on [Petr Github][].k
+
+[Petr Github]: https://github.com/PetrKryslUCSD/HowToUseJuliaWithSublimeText3/tree/master/snippets
+
+## Usage
+
+### Open terminal
+
+Bring up the **Command Palette**, start typing in `Terminus`.
+Select **Terminus: List Shells**,
+and from the list that appears choose the shell you wish to start.
+Note that then you can select whether to start the shell
+in a panel at the bottom or in a separate view.
+
+Julia may be started in the resulting terminal in the usual way.
+The terminals that I have checked out (`cmd` and the WSL Ubuntu shell)
+work as expected.
+
+### Open a source file and then run Julia from the source file
+
+Open a Julia source file,
+and in the **Command Palette**, start typing in `Terminus`.
+Select **Terminus: Open Julia**.
+This will open the default terminal on the user's platform
+(`cmd` on Windows, but see also the note about the Git bash below),
+and run Julia in the directory that contains the open file.
+
+### Evaluating code
+
+Select some Julia code and press <Key k="ctrl+enter" />.
+The code will be pasted into the **Terminus** window
+and evaluated in Julia (assuming Julia was started in that **Terminus** window).
+This also works for evaluating a line of code:
+place the cursor on a line and type <Key k="ctrl+enter" />.
+
+#### Running Julia files
+
+In a opened Julia source file, press <Key k="ctrl+b" />
+(which is the key binding for **Tools → Build**).
+The current file will be evaluated in a Julia-running Terminus window
+with an `include()`.
+
+### Asking for help in the REPL
+
+From a Julia source file, press <Key k="ctrl+shift+x, ctrl+h"> to ask the Julia
+REPL to display help about the current selection.
+
+## Zeal
+
+Zeal is an open-source browser of off-line documentation.
+There is a Sublime package that allows you to jump to Zeal documentation
+for the symbol under cursor.
+Zeal can also be used with other languages than Julia.
+Zeal can be downloaded from [Zeal website][].
+The documentation for Julia needs to be downloaded within the Zeal documentation browser:
+in the menu "Tools" choose "Docsets".
+
+From Package Control install [Zeal for Sublime Text][].
+
+The Zeal executable needs to be revealed to the editor.
+Also, Zeal
+needs to be made aware of the language of the documentation request:
+
+```json
+{
+  /**
+   Zeal executable path.
+   The instructions below require zeal
+   to be located very precisely by its full path!?
+   */
+  "zeal_command": "C:\\Program Files\\Zeal\\zeal.exe",
+  "mapping_sort" : true,
+
+  // Language mapping.
+  "language_mapping": {
+    "Python": {"lang": "python", "zeal_lang": "python"},
+    "Julia": {"lang": "julia", "zeal_lang": "julia"}
+  }
+}
```

Comment:
These settings use the old format for docset specification. I rewrote the package a few months ago and the settings now [look like this](https://github.com/vaanwd/Zeal/blob/master/Zeal.sublime-settings). You could either submit a pull request to add Julia to the default set, or suggest to change the settings to include `"docsets_user": [ { "name": "Julia" } ],` (or `"fallback": "guess"`).